src/LowMachEquationSystem.C

```diff
@@ -165,6 +165,9 @@
 #include <user_functions/KovasznayVelocityAuxFunction.h>
 #include <user_functions/KovasznayPressureAuxFunction.h>
 
+#include <user_functions/DropletVelocityAuxFunction.h>
```

Comment:
Any reason for the ordering here? Want to group the multiphase aux functions together?